### PR TITLE
Add audit board members to audit report

### PR DESCRIPTION
**Pull request by jonahkagan**
**DESCRIPTION**

Task: #237 
Adds a row to the audit board report with each audit board member's name and party affiliation, as well as the name of the audit board they were on.

**New Tests**

Adds test_report.py with a simple test that also covers the rest of the report content.

**Checklist**

*Function*

[] Code is fully functional and ready for review (or is tagged WIP)

[] No errors/warnings in the browser console ( ﾟヮﾟ)/

[] package.json is committed if I added or removed any dependencies

[] Tests added as needed to ensure code base still has 100% passing coverage

[] Tests added as needed to cover new functionality

*Style*

[] Passes linter without errors

[] Code is easily understandable and self-documenting (using naming conventions in keeping with the rest of the codebase)

[] Interfaces all have 'I' before their names (as in `IAudit`)

[] Comments are only included as needed to explain the 'why' instead of the 'what'

[] Common standards and coding practices are adhered to

[] DRY: Don't Repeat Yourself. Code is reasonably structured to avoid repetition

[] Code is intuitively structured following a Separation of Concerns

[] No class components; only functional components (using Hooks as needed)

*Workflow*

[] Do an interactive rebase or a rebase

### CODE REVIEW

File: arlo_server/__init__.py [hunk 1]
```diff
@@ -841,6 +841,16 @@ def audit_report(election_id):
     report_writer.writerow(["Risk Limit", "{:d}%".format(election.risk_limit)])
     report_writer.writerow(["Random Seed", election.random_seed])
 
+    for audit_board in jurisdiction.audit_boards:
+        report_writer.writerow([
+            audit_board.name, audit_board.member_1,
```
Comment (benadida) [inline arlo_server/__init__.py]: this highlights that an audit board name might not be the nice "Audit Board #1", and coud bite us later. OK for now, I don't have a better solution.

File: tests/test_app.py [hunk 2]
```diff
@@ -77,6 +77,28 @@ def test_whole_audit_flow(client):
     assert result2["riskLimit"] == 5
 
 
+def setup_audit_board(client, election_id, jurisdiction_id, audit_board_id):
```
Comment (benadida) [inline tests/test_app.py]: nice, thanks for adding this.

File: arlo_server/__init__.py (hunk 1 above)
Comment (jonahkagan) [reply arlo_server/__init__.py]: I'm not sure what you mean - I was just going off the data model, which has a name field. What might bite us?

Comment (benadida) [reply arlo_server/__init__.py]: I mean that if the audit board has a name of "Foobar", then the row in the CSV won't be clear that it's an audit board cause it has no title. Right now the audit boards have nice names, so it works okay.

For example, the previous line in the report says "Random Seed" explicitly.

**Pull request merged**